example/simple-server/simple-server.go

```diff
@@ -58,17 +59,31 @@ func plain(listenAddr string) {
 	serve(listener, "plain-internet")
 }
 
-func withZiti(service string) {
+func zitifiedServer() {
 	options := ziti.ListenOptions{
 		ConnectTimeout: 5 * time.Minute,
 		MaxConnections: 3,
 	}
-	listener, err := ziti.NewContext().ListenWithOptions(service, &options)
+
+	// Get identity config
+	cfg, err := config.NewFromFile(os.Args[1])
+	if err != nil {
+		panic(err)
+	}
+
+	// Get service name (defaults to "simpleService")
+	serviceName := "simpleService"
+	if len(os.Args) > 2 {
+		serviceName = os.Args[2]
```

Comment:
would you mind adding a log message here that says it's using the default, and the default is 'simpleService'?

Reply:
Would you want this to be the actual pfx log (used if DEBUG is set to true) or just a stdout print statement?

If it's the latter, we do have this print out before the server is started 
```
fmt.Printf("listening for requests for Ziti service %v\n", serviceName)
```

Reply:
probably pfxlog. it's already imported and initialized. might as well use it

Reply:
and i'd just normalize it all to one or the other... i don't care terribly which way it goes